build.sbt

```
@@ -19,21 +22,12 @@ inThisBuild(Seq(
   scalacOptions ++= Seq(
     "-deprecation",
     "-Xfatal-warnings",
-    "-encoding", "UTF-8"
+    "-encoding", "UTF-8",
+    "-release:11",
   ),
   // sonatype metadata
   organization := "com.gu",
   licenses := Seq("Apache V2" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")),
```

This can be [tweaked](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/configuration.md#recommended-sbt-settings) to:

```suggestion
  licenses := Seq(License.Apache2),
```

---

Yeah thanks, might as well update that now as well 👍